LocalSettings.php

```diff
@@ -2069,10 +2069,13 @@
 	// Kartographer
 	'wgKartographerWikivoyageMode' => [
 		'default' => false,
-	 ],
+	],
 	'wgKartographerUseMarkerStyle' => [
 		'default' => false,
-	 ],
+	],
+	'wgKartographerMapServer' => [
+		'default' => 'https://a.tile.openstreetmap.org',
```

Comment:
I thought OSM's advice was don't use a specific server?

Reply:
Oh, that might actually work. will remove `a.` then.